client/app/account/billing/payment/method/billing-payment-method.controller.js

```diff
@@ -41,6 +43,20 @@ export default class BillingPaymentMethodCtrl {
   =            EVENTS            =
   ============================== */
 
+  /**
+   *  Navigate to add payment method
+   */
+  goToAddPayment() {
+    this.atInternet.trackClick({
```

Comment:
Doesn't this return a `Promise` ?

Reply:
No it doesnt.
https://github.com/ovh-ux/ng-at-internet/blob/master/src/provider.js#L345